src/components/FeeSelector/index.tsx

```diff
@@ -145,6 +189,14 @@ function FeeSelector({
   currencyB: Currency | undefined
 }) {
   const theme = useTheme()
+  const { chainId } = useActiveWeb3React()
+
+  const FEE_AMOUNTS = [ChainId.ARBITRUM, ChainId.FANTOM, ChainId.BTTC].includes(chainId)
+    ? ALL_FEE_AMOUNTS
+    : [FeeAmount.VERY_STABLE, FeeAmount.VERY_STABLE1, FeeAmount.STABLE, FeeAmount.MOST_PAIR2, FeeAmount.EXOTIC]
```

Comment:
Should this be memo?

Reply:
```
  const poolAddresses = useProAmmPoolInfos(currencyA, currencyB, FEE_AMOUNTS)
```
`FEE_AMOUNTS` likely to cause this line 220 to re-render everytime it runs

Reply:
```
    // eslint-disable-next-line react-hooks/exhaustive-deps
    [currencyA, currencyB, proAmmCoreFactoryAddress, JSON.stringify(feeAmount), networkInfo],
```

you added this to that hook. did you remember?

Reply:
:))) nope